### DISCUSSION TITLE
Please allow (non-deepdiver) level 10 hull upgrades to resist the crush depth in the endlocation

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#4931] [Ideas and suggestions]
The current end location causes continuous hull damage before you can reach the ending with non-deepdiver submarines, paired with not being able to transfer upgrades (which requires massive cash investment) and "pushed" to use a particular class of submarines result in some amount of disappointment when it is revealed (due to fog of war).

Thank you, please do consider.

**Comment by NilanthAnimosus** [comment]
Given as far as I am aware - the other classes do not even do anything mechanically speaking and its practically a flavour text, this is a very weird thing to have that submarine type is a restriction to actually complete / reach the end of the game.

Perhaps this would make sense if submarines had other functionality attached to them, but honestly making attack submarines fire coilguns magically better to balance it just makes that sound weirdly worse than just allowing all submarines to reach the same point eventually.

Can all submarines get the same depth upgrades and same maximum depth, to not have the above (if it is an issue I've never reached the end) be an issue, instead have class effect what upgrades cost? so a none deep diver pays 2x to 8x the price for depth upgrades. this would seem more reasonable to me.

**Comment by Udrakan** [comment]
This seems like the submarine class system was implemented too early, without some other features that will add depth to it later. I wish it was pushed back to time when those other features get in the game.

**Comment by MerelyMezz** [comment]
Pressure as it is implemented right now was the single most unenjoyable mechanic our crew had to deal with in the entire game. We played on the same sub for the better part of a year, over the course of which we learned to face and deal with increasingly difficult and hazardous obstacles and making many great memories along the way. Alot of things were really tough to deal with, but with the right amount of preparation and organization they were also overcomable, however pressure was not like that. It's in essence an arbitrary stop sign that feels shitty and intrusive. Even with the entire crew being dedicated to welding duties it was straight up impossible to drain the ship faster than it was flooding, even on our tiny shuttle with a large surplus of welding fuel available and very little amounts of hull that needed attention.

I honestly don't really care for "deep divers" being a thing, every sub should be principally able to make it to the end. Not having selected the "correct" type of submarine* for the end isn't something that should get in the player's way. We spent alot of time on our submarine and got quite attached to it, being told that we can't use it for the end is simply UN-AC-CEPT-ABLE. It's bad enough that to even continue into the deep sea you need to grind heaps of money for all ten of the hull upgrades**, making them in essence the only mandatory upgrade of the entire game, which isn't really properly communicated until it's too late, but being forced to abandon the ship that we spent a total of 70+ hours on together because of some designer's misguided applications of "r e a l i s m" is by a wide margin the most disappointing resolution to a game I've invested alot of time in. It would have been preferable outcome to have had the whole crew die to a ludicrously dangerous threat that we atleast would have had a feasible arsenal of strategies to use against beyond simply "use a deep diver DUH".

----
\* especially if literally all it is, is a sticker you slap on in the sub editor
\*\* which to be honest we just sidestepped with a save editor, because there's no way we were going through a massive grind like that when we could only reliably meet to play once a week
